src/Uno.UI.Runtime.Skia.Wpf/Uno.UI.Runtime.Skia.Wpf.csproj

```diff
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
 	<PropertyGroup>
-		<TargetFrameworks>net7.0-windows</TargetFrameworks>
-		<TargetFrameworks Condition="'$(UnoDisableNet8)'!='true'">$(TargetFrameworks);net8.0-windows</TargetFrameworks>
+		<TargetFrameworks>$(NetPrevious)-windows</TargetFrameworks>
+		<TargetFrameworks Condition="'$(UnoDisableNet8)'!='true'">$(TargetFrameworks);$(NetCurrent)-windows</TargetFrameworks>
```

Comment:
Why is this logic here, rather with the other logic about previous and current versions?
Also, this indirectly ties `UnoDisableNet8` and `NetCurrent` - which won't always be the case. - Can this be avoided?
What happens here where NetCurrent isn't the same as Net8?

Reply:
I think the property should be named `UnoDisableNetCurrent`

This part is essentially a workaround for a long-standing .NET SDK bug (https://github.com/dotnet/sdk/issues/30103)